server/.golangci.yml

```diff
@@ -103,7 +103,7 @@ issues:
         channels/api4/usage_test.go|\
         channels/api4/user.go|\
         channels/api4/user_local.go|\
-        channels/api4/user_test.go|\
+        channels/api4/webhook.go|\
```

Comment:
That seems wrong

Reply:
This file got merge conflicts to solve it had to add this.